File: experiment/java-python/test/j2pyraw/if__0__BASIC_AND_EXTENDED_EUCLIDEAN_ALGORITHMS_1.java
class X { public static int gcdExtended ( int a , int b , int x , int y ) { int x1 = 1 , y1 = 1 ; int gcd = gcdExtended ( b % a , a , x1 , y1 ) ; x = y1 - ( b / a ) * x1 ; y = x1 ; return gcd ; }
 }

File: experiment/java-python/test/j2pyrawpy/if__0__BASIC_AND_EXTENDED_EUCLIDEAN_ALGORITHMS_1.py
#!/usr/bin/env python
""" generated source for module if__0__BASIC_AND_EXTENDED_EUCLIDEAN_ALGORITHMS_1 """
class X(object):
    """ generated source for class X """
    @classmethod
    def gcdExtended(cls, a, b, x, y):
        """ generated source for method gcdExtended """
        x1 = 1
        y1 = 1
        gcd = cls.gcdExtended(b % a, a, x1, y1)
        x = y1 - (b / a) * x1
        y = x1
        return gcd

